source/usage-examples/changeStream.txt

```diff
@@ -88,17 +88,17 @@ print change events that occur on the collection.
 
 First, open the change stream on the collection and then define a callback
 on the change stream using the ``on()`` method. Once set, generate a change
-event to be emitted by performing a change to the collection.
+event to emit by performing a change to the collection.
```

Comment:
```suggestion
event by performing a change to the collection.
```

Suggestion: Im not sure "to emit" is needed here, and it sounds odd that the change event was generated be "emitted", but rathe it was just generated.